release-tools/mktarball.sh

```diff
@@ -6,11 +6,30 @@ if [ -z $VERSION ]; then
 	exit 1
 fi
 DESTDIR=tarsnap-autoconf-${VERSION}
+RELEASEDATE=`date "+%B %d, %Y"`
 
 # Copy bits in
 mkdir ${DESTDIR}
 cp COPYING Makefile.am actarsnap.m4 configure.ac tsserver .autom4te.cfg ${DESTDIR}/
 cp -R keygen keymgmt keyregen lib libarchive libcperciva misc pkg recrypt tar ${DESTDIR}/
+# Copy with substitution
+for f in \
+	keymgmt/tarsnap-keymgmt.1-man \
+	keymgmt/tarsnap-keymgmt.1-mdoc \
+	keyregen/tarsnap-keyregen.1-man \
+	keyregen/tarsnap-keyregen.1-mdoc \
+	keygen/tarsnap-keygen.1-man \
+	keygen/tarsnap-keygen.1-mdoc \
+	tar/tarsnap.1-man.in \
+	tar/tarsnap.1-mdoc.in \
+	tar/tarsnap.conf.5-man.in \
+	tar/tarsnap.conf.5-mdoc.in \
+	recrypt/tarsnap-recrypt.1-man \
+	recrypt/tarsnap-recrypt.1-mdoc \
```

Comment:
How about `*/*-man */*-mdoc */*-man.in */*-mdoc.in` here instead of listing all the files individually?